docs/tokenomics/nns/nns-intro.md

```diff
@@ -1,218 +1,53 @@
-# NNS intro
+# Network Nervous System (NNS)
 
-The NNS is the decentralized autonomous organization (DAO) that governs the Internet Computer Protocol. It is a fully on-chain, decentralized organization and is, for instance, responsible for making protocol level upgrades to continuously improve the Internet Computer Protocol. It does this via an implementation of liquid democracy in which ICP neuron holders vote on proposals that shape the development of the Internet Computer. Once such a proposal is accepted, it is autonomously executed.
+## Overview​
+The [NNS](https://nns.ic0.app/) is the decentralized autonomous organization (DAO) that governs the Internet Computer Protocol. 
 
-While other blockchains take weeks or months to upgrade (sometimes called hard fork) and typically require substantial manual work and coordination to do so, ICP upgrades itself on a weekly basis (https://dashboard.internetcomputer.org/releases). Its ability to upgrade and iterate is a comparative "superpower."
+It is a fully on-chain, decentralized organization and is, for instance, responsible for making protocol-level upgrades to continuously improve the Internet Computer Protocol. 
 
-The NNS works by allowing users to stake ICP governance tokens to create voting neurons. Anyone can create a neuron, which together exert the will of the community, mediated through algorithms. Neurons are like accounts where notice of withdrawals must be given, where the length of the notice period is a configuration known as the “dissolve delay.” The voting power of neurons, and their relative claim to voting rewards, is proportional to the quantity of staked ICP, the length of the configured dissolve delay, and their “age.” Neurons can be made to vote manually, or automatically, by following other neurons in a form of liquid democracy.
+It operates under a liquid democracy in which those who have staked their ICP or [ICP neuron holders vote](../../concepts/governance.md) vote on proposals that shape the development of the Internet Computer. 
 
-Neuron holders are placed in a crypto-economic game, where they are incentivized to vote to adopt and reject proposals, or to configure neuron follows that cause them to vote automatically in a desirable way, according to what is most likely to drive the value of the Internet Computer protocol over the long term.
+Examples of proposals include:
+1. upgrade the protocol and software used by the node machines that host the network
+2. create new subnet blockchains to increase network capacity
+3. split subnets to divide their load
+4. configure economic parameters that control how much must be  paid by users for compute capacity
+5. In extreme situations, it can freeze malicious canister smart contract software to protect the network; and many other things. 
```

Comment:
do we really want to point out this controversial topic? I would rather leave that out.

If you want one more once could mentioned the proposals that update the DAO itself (with voting rules etc)